neptune/query/query.go

```diff
@@ -84,4 +84,13 @@ const (
 
 	// dimension
 	CreateDimensionToInstanceRelationship = "g.addV('_%s_%s').property('value','%s').as('d').addE('HAS_DIMENSION').from(V().hasLabel('_%s_Instance')).select('d').by(id)"
+
+	// observation
+	GetInstanceHeader      = "g.V().hasLabel('_%s_Instance').as('instance')"
+	GetAllObservationsPart = ".V().hasLabel('_%s_observation').values('row')"
+
+	GetObservationsPart         = ".V().hasLabel('_%s_observation').match("
+	GetObservationDimensionPart = "__.as('row').out('isValueOf').hasLabel('_%s_%s').where(values('value').is(within(%s)))"
+	GetObservationSelectRowPart = ".select('instance', 'row').by('header').by('row').unfold().dedup().select(values)"
+	LimitPart                   = ".limit(%v)"
```

Comment:
```suggestion
	LimitPart                   = ".limit(%d)"
```

So that go lint can give build-time type mismatch warnings.